scram-tools.file/tools/xrootd/xrootd.xml

```diff
@@ -1,4 +1,4 @@
-<tool name="xrootd" version="@TOOL_VERSION@">
+<tool name="xrootd" version="@TOOL_VERSION@" revision="1">
```

Comment:
@iarspider , I think scram in 13.0.X does not support revision so please remove `revision="1"`

Reply:
ok